### PR TITLE
PS-7779 Document system-wide crypto-policies for RHEL/CentOS (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.7 version
fixed conflict UU source/installation/yum_repo.rst

### CODE REVIEW

File: doc/source/installation/yum_repo.rst [hunk 1]
```diff
@@ -215,6 +218,7 @@ The following commands start, provide the server status, stop the server, and re
 
 .. note::
 
+<<<<<<< HEAD
```
Comment (alina-derkach-oaza) [inline doc/source/installation/yum_repo.rst]: Should this be deleted?

File: doc/source/installation/yum_repo.rst [hunk 2]
```diff
@@ -225,6 +229,9 @@ Working with SELinux
 ======================
 
 For information on working with SELinux, see :ref:`selinux`.
+=======
+  The *RHEL* 8 distributions and derivatives have added `system-wide cryptographic policies component <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening>`__. This component allows the configuration of cryptographic subsystems. 
+>>>>>>> pb/ps-5.7-7779
```
Comment (alina-derkach-oaza) [inline doc/source/installation/yum_repo.rst]: Should this be deleted?

File: doc/source/installation/yum_repo.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst]: Updated.

File: doc/source/installation/yum_repo.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/installation/yum_repo.rst]: No.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**